app/views/home/index.html.erb

```diff
@@ -5,9 +5,13 @@
       <li>
           <%= link_to post.title, post.url %>
           <br>
+          <br>
           Posted by <%= link_to post.user.name, post.user %>  
-          <%= time_ago_in_words(post.created_at) %> ago. 
+          <%= time_ago_in_words(post.created_at) %> ago.
+          <br>
+          <%= link_to "comments", post_path(post.id) %> 
```

Comment:
would it be possible to show the number of comments here? could make use of the built in Rails [pluralize](http://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize) helper method.